cmd/cloudFoundryDeploy.go

```diff
@@ -724,18 +724,20 @@ func handleMtaExtensionCredentials(extFile string, credentials map[string]interf
 	for name, credentialKey := range credentials {
 		credKey, ok := credentialKey.(string)
 		if !ok {
-			return fmt.Errorf("Cannot handle mta extension credentials: Cannot cast '%v' (type %T) to string", credentialKey, credentialKey)
+			return fmt.Errorf("cannot handle mta extension credentials: Cannot cast '%v' (type %T) to string", credentialKey, credentialKey)
 		}
-		pattern := "<%= " + name + " %>"
-		if strings.Contains(content, pattern) {
+		pattern := regexp.MustCompile("<%=\\s*" + name + "\\s*%>")
```

Comment:
We trust that `name` does not contain parts which are interpreted by regex (... eg `.*`). Don't know how we can avoid that ....